src/components/shared/Extensions/Accordion/Accordion.tsx

```diff
@@ -24,7 +24,7 @@ const Accordion: FC<AccordionProps> = ({
   };
 
   return (
-    <div className="bg-base-white">
```

Comment:
are you sure about this change?

Reply:
As I can see this component is used only in LazyConsensusPage and works correctly